package.json

```diff
@@ -53,7 +53,7 @@
     "core-js": "3.37.0",
     "del": "6.1.1",
     "dtslint": "4.2.1",
-    "eslint": "8.57.0",
+    "eslint": "^8.57.0",
```

Comment:
Can you please revert the changes in `package.json` and `package-lock.json`?

Reply:
My apologies I am pretty new to lint

Reply:
👍 np